cpp/src/detail/utility_wrappers.cu

```diff
@@ -227,5 +229,36 @@ template bool is_equal(raft::handle_t const& handle,
                        raft::device_span<int64_t const> span1,
                        raft::device_span<int64_t const> span2);
 
+template <typename data_t, bool multi_gpu>
+size_t count_values(raft::handle_t const& handle,
+                    raft::device_span<data_t const> span,
+                    data_t value)
+{
+  size_t local_count = thrust::count_if(
+    handle.get_thrust_policy(), span.begin(), span.end(), [value] __device__(data_t d) {
+      return d == value;
+    });
```

Comment:
You can use thrust::count (https://thrust.github.io/doc/group__counting_ga4f2955d87ada73fc89f3442a19ca1bfa.html#ga4f2955d87ada73fc89f3442a19ca1bfa) and no need to provide a predicate.

Reply:
Fixed.